site/content/en/docs/tasks/manage/setup_sequential_admission.md

```diff
@@ -93,6 +94,15 @@ If you don't specify any value for `backoffLimitCount`,
 a Workload is repeatedly and endlessly re-queued to the queue based on the `timestamp`.
 Once the number of re-queues reaches the limit, Kueue [deactivates the Workload](/docs/concepts/workload/#active).
 
+{{% alert title="Warning" color="warning" %}}
+_The `backoffBaseSeconds` is available in Kueue v0.7.0 and later_
+{{% /alert %}}
+The time to re-queue a workload after each consecutive timeout is increased
+exponentially, with the exponent of 2. The first delay is determined by the
+`backoffBaseSeconds` parameter (defaulting to 10). There is additionally a small
+random jitter added to the time to re-queue. So, after the consecutive timeouts
```

Comment:
Remove the jitter bit. It's TMI for users.

Reply:
Done